website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Launch Templates and Custom Images"

```

Comment:
I'd just call this `LaunchTemplates`

Reply:
I'd like to offer a bread crumb to non-AWS people. Do you think it's intuitive that "custom node image" is covered by "launch template"?